vocabularies/geological-sites.ttl

```diff
@@ -138,32 +143,69 @@ gsqst:colluvial a skos:Concept ;
 
 gsqst:outcrop a skos:Concept ;
     skos:broader gsqst:field-site ;
-    dcterms:source "Merriam-Webster Dictionary" ;
+    dct:source "Merriam-Webster Dictionary" ;
     skos:definition "The part of a rock formation that appears at the surface of the ground."@en ;
     skos:inScheme <https://linked.data.gov.au/def/geological-sites> ;
     skos:notation "oc" ;
     skos:prefLabel "Outcrop"@en .
 
-gsqst:soil-horizon a skos:Concept ;
-    skos:altLabel "Soil"@en ;
+gsqst:regolith a skos:Concept ;
     skos:broader gsqst:field-site ;
+    skos:definition "A field site at a layer of loose, heterogeneous superficial deposits covering solid rock."@en ;
+    skos:inScheme <https://linked.data.gov.au/def/geological-sites> ;
+    skos:notation "rg" ;
+    skos:prefLabel "Regolith"@en .
+
+gsqst:pedolith a skos:Concept ;
+    skos:altLabel "Soil"@en,
+        "Soil Horizon"@en ;
+    skos:broader gsqst:regolith ;
+    skos:definition "A field site that has undergone pedogenic processes. Typically where a sample is taken or observation is made of soil."@en ;
+    skos:inScheme <https://linked.data.gov.au/def/geological-sites> ;
+    skos:notation "pd" ;
+    skos:prefLabel "Pedolith"@en .
+
+gsqst:saprolith a skos:Concept ;
+    skos:altLabel "Decomposed Granite"@en,
+        "Decomposed Rock"@en,
+        "Weathered Rock"@en,
+        "Weathering Profile"@en ;
+    skos:broader gsqst:regolith ;
     skos:definition "A field site from which a soil sample is taken or observation is made."@en ;
     skos:inScheme <https://linked.data.gov.au/def/geological-sites> ;
-    skos:notation "sh" ;
-    skos:prefLabel "Soil Horizon"@en .
+    skos:notation "sr" ;
+    skos:prefLabel "Saprolith"@en .
+
+gsqst:subcrop a skos:Concept ;
+    skos:altLabel "sub-outcrop"@en ;
+    skos:broader gsqst:field-site ;
+    skos:definition "The part of a rock formation under soil or sediment cover that is close to, but not outcropping at, the ground surface."@en ;
```

Comment:
def needs to be replaced.
suggested def "A field site where bedrock is only exposed over a small area with very little topographic relief, to such an extent that the bedrock is reasonably inferred to be in-situ. There is a reasonable degree of confidence that samples and observations from a subcrop site relate to the bedrock at that location. "

Reply:
done